src/main/java/decodex/data/exception/ParserException.java

```diff
@@ -2,14 +2,13 @@
 
 public class ParserException extends Exception {
 
-    public static final String MISSING_COMMAND_ARGS_MESSAGE = "Your command arguments is empty."
-            + " Please try again after tone...BEEP";
-    public static final String MISSING_COMMAND_TYPE_MESSAGE = "Command cannot be empty!!";
+    public static final String MISSING_COMMAND_ARGS_MESSAGE = "[-] Your command arguments is empty";
+    public static final String MISSING_COMMAND_TYPE_MESSAGE = "[-] Command cannot be empty";
 
     /**
      * This error message is dedicated to users intending to find bugs in our program.
      */
-    public static final String WEIRD_COMMAND_TYPE_MESSAGE = "In what universe did you think that would break us."
+    public static final String WEIRD_COMMAND_TYPE_MESSAGE = "[x] In what universe did you think that would break us."
```

Comment:
As this message is used when the number of tokens is less than expected, it is not very useful when genuinely missing parameters. Consider changing the message and variable name?